tests/basic/basic_test.go

```diff
@@ -291,13 +294,14 @@ var _ = Describe("{AppTasksDown}", func() {
 	})
 })
 
-func getMaxNodesToBeDown(replicas int) int {
+func getMaxNodesToBeDown(nodes, replicas int) int {
 	if replicas == 1 {
 		return 0
 	}
-	if replicas%2 != 0 {
+	if nodes > 3 && replicas%2 != 0 {
```

Comment:
this will shutdown 2 nodes in a 4 node cluster for a repl 3 volume and hence will bring PX out of quorum.

Also add a comment on `getMaxNodesToBeDown` on the purpose of the function.